site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,459 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Optimizing LCP using Signed Exchanges
+description: >
+  Signed exchanges are a means for web developers to drastically improve page loading speed coming from SXG referrers such as Google Search. They can provide benefit simply from being enabled by a checkbox, but you can take additional steps to get the most out of them.
+subhead: >
+  How to measure and optimize signed exchanges to get the most improvement out of them.
+date: 2022-04-15
+
+authors:
+  - twifkak
+
+tags:
+  - performance
+
+# Optional
+# Hero images should be at least 1920 x 960
+# https://developer.chrome.com/docs/handbook/how-to/add-media/
+# hero: 'image/BrQidfK9jaQyIHwdw91aVpkPiib2/EnMzOm0mBytBA3AzlCG6.png'
+
+# Optional
+# You can provide an optional cropping of your hero image to be used as a
+# thumbnail. Note the alt text will be the same for both the thumbnail and
+# the hero.
+# Thumbnail images should be at least 828 x 416
+# thumbnail: 'image/BrQidfK9jaQyIHwdw91aVpkPiib2/EnMzOm0mBytBA3AzlCG6.png'
+
+# Required if there is a hero or social image
+# alt: An alternative text description of your hero or social image.
```

Comment:
```suggestion
```

Reply:
Just removing some boilerplate here. If you need to add hero images, let me know. I can edit them. Otherwise, we're fine to publish without them.